index.html

```diff
@@ -6,10 +6,130 @@
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
   <link rel="stylesheet" href="style.css">
   <title>Portfolio</title>
 </head>
 <body>
-  
+  <nav class="toolbar">
+    <a href="#" class="logo">Welcome</a>
+    <a href="#" class="menu-icon fa-solid fa-bars"></a>
+  </nav>
+  <section class="header">
+  </section>
+  <section class="headline">
+    <h1 class="headline-top1">Hey there. I'm Sergiy</h1>
+    <h1 class="headline-top2">I'm a software developer</h1>
+    <p class="headline-text"> 
+      I can help you build a product ,
+      feature or website Look through
+      some of my work and experience!
+      <br>
+      If you like what you see and have
+      a project you need coded, 
+      dont hestiate to contact me.
+    </p>
+    <ul class="nobull links">
+      <li>
+        <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
+      </li>
+    </ul>
+    <p class="button-normal">
+      <img src="./img/button-normal.png" alt="">
+    </p>
+  </section>
+  <section class="my-recent-works">My Recent Works</section>
```

Comment:
- [ ] The title (e.g., "My Recent Works") uses a header tag hx instead of the section to make your portfolio structure well.